install.ts

```diff
@@ -0,0 +1,81 @@
+import { parse } from "https://deno.land/std@0.136.0/flags/mod.ts";
+import { red } from "https://deno.land/std@0.136.0/fmt/colors.ts";
+
+/*
+ * Prompt for a response
+ */
+async function prompt(message = "") {
+  const buf = new Uint8Array(1024);
+  await Deno.stdout.write(new TextEncoder().encode(message + ": "));
+  const n = <number> await Deno.stdin.read(buf);
+  return new TextDecoder().decode(buf.subarray(0, n)).trim();
+}
```

Comment:
```suggestion
```

There's no need for a custom `prompt` implementation. Deno aims to implement most Web APIs verbatim, so it supports `prompt` natively: https://doc.deno.land/deno/stable/~/prompt